python/ray/rllib/a3c/envs.py

```diff
@@ -34,13 +35,27 @@ def _process_frame42(frame):
     return frame
 
 
+def _process_frame80(frame):
+    frame = frame[34:(34 + 160), :160]
+    # Resize by half, then down to 42x42 (essentially mipmapping). If we resize
+    # directly we lose pixels that, when mapped to 42x42, aren't close enough
+    # to the pixel boundary.
+    frame = cv2.resize(frame, (80, 80))
+    frame = frame.mean(2)
+    frame = frame.astype(np.float32)
+    frame *= (1.0 / 255.0)
+    frame = np.reshape(frame, [80, 80, 1])
+    return frame
+
+
 class AtariProcessing(gym.ObservationWrapper):
     def __init__(self, env=None):
         super(AtariProcessing, self).__init__(env)
-        self.observation_space = Box(0.0, 1.0, [42, 42, 1])
+        # self.observation_space = Box(0.0, 1.0, [42, 42, 1])
```

Comment:
Let's remove this comment. Unless you think we may comment it back in at some point, in which case let's add a comment explaining when we would want to do that.